LoRaEngine/LoraKeysManagerFacade/DeviceGetter.cs

```diff
@@ -91,7 +93,7 @@ public async Task<List<IoTHubDeviceInfo>> GetDeviceList(string devEUI, string ga
                 var cacheKeyDevNonce = string.Concat(devEUI, ":", devNonce);
                 var lockKeyDevNonce = string.Concat(cacheKeyDevNonce, ":joinlockdevnonce");
```

Comment:
FYI, noticed that this is unused and opened PR #1070 to remove it as it's unrelated to this.

Reply:
Thanks, will take a look!